code/game/objects/structures/guillotine.dm

```diff
@@ -77,19 +77,19 @@
 	var/msg = "It is [anchored ? "wrenched to the floor." : "unsecured. A wrench should fix that."]<br/>"
 
 	if (blade_status == GUILLOTINE_BLADE_RAISED)
-		msg += "The blade is raised, ready to fall, and"
+		msg += span_notice("The blade is raised, ready to fall, and")
 
 		if (blade_sharpness >= GUILLOTINE_DECAP_MIN_SHARP)
-			msg += " looks sharp enough to decapitate without any resistance."
+			msg += span_notice(" looks sharp enough to decapitate without any resistance.")
 		else
-			msg += " doesn't look particularly sharp. Perhaps a whetstone can be used to sharpen it."
+			msg += span_notice(" doesn't look particularly sharp. Perhaps a whetstone can be used to sharpen it.")
 	else
-		msg += "The blade is hidden inside the stocks."
+		msg += span_notice("The blade is hidden inside the stocks.")
 
 	. += msg
```

Comment:
Remember, we don't need the `<br/>` anymore on the `"It is [anchored ? "wrenched to the floor." : "unsecured. A wrench should fix that."]"` because now it will make the examine box much uglier. do remove that.